src/hooks/common/use-app-title.ts

```diff
@@ -4,13 +4,16 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import { useEffect } from 'react'
+import { useMemo } from 'react'
 import { useApplicationState } from './use-application-state'
 
-export const useDocumentTitle = (title?: string): void => {
+/**
+ * Returns the app title with branding if set.
+ */
+export const useAppTitle = (): string => {
   const brandingName = useApplicationState((state) => state.config.branding.name)
 
-  useEffect(() => {
-    document.title = `${title ? title + ' - ' : ''}HedgeDoc ${brandingName ? ` @ ${brandingName}` : ''}`
-  }, [brandingName, title])
+  return useMemo(() => {
+    return `HedgeDoc ${brandingName ? ` @ ${brandingName}` : ''}`
```

Comment:
```suggestion
    return `HedgeDoc${brandingName ? ` @ ${brandingName}` : ''}`
```

There is a space too much

Reply:
![image](https://user-images.githubusercontent.com/52606896/147883407-b69a793b-3eac-4f06-b486-076202febee7.png)